pnpm-lock.yaml

```diff
@@ -1,399 +1,1656 @@
-lockfileVersion: '6.0'
+lockfileVersion: '9.0'
```

Comment:
Probably this should be a separate PR?

Reply:
Agreed. I've committed this change by mistake. Thanks!